lab_1/main.py

```diff
@@ -125,4 +234,18 @@ def sort_concordance(tokens: list, word: str, left_context_size: int, right_cont
     left_sort = True
     --> [['dog', 'is', 'happy', 'but', 'the', 'cat'], ['man', 'is', 'happy', 'the', 'dog', 'is']]
     """
-    pass
+    if type(tokens) == list and type(word) == str and type(left_context_size) == int and type(right_context_size) == int and type(left_sort) == bool:
```

Comment:
Usually, there are check for input arguments and body function - on the same level.
No need for nesting.
Also, `type`.